packages/store/src/Utils.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+library Utils {
+  function divCeil(uint256 a, uint256 b) internal pure returns (uint256) {
+    return a / b + (a % b == 0 ? 0 : 1);
+  }
+
+  /**
+   * Adapted from https://github.com/dk1a/solidity-stringutils/blob/main/src/utils/mem.sol#L149-L167
+   * @dev Left-aligned bit mask (e.g. for partial mload/mstore).
+   * For length >= 32 returns type(uint256).max
+   *
+   * length 0:   0x000000...000000
+   * length 1:   0xff0000...000000
+   * length 2:   0xffff00...000000
+   * ...
+   * length 30:  0xffffff...ff0000
+   * length 31:  0xffffff...ffff00
+   * length 32+: 0xffffff...ffffff
```

Comment:
Since you changed it from bytes to bits the comments are confusing.
But it should probably be inlined into `_loadPartialWord` and `_storePartialWord` and combined with the other bit manipulation for efficiency anyways

Reply:
True, removed the comments for now, but agree with inlining it later